ui/src/tree/AzExtTreeItem.ts

```diff
@@ -116,6 +117,15 @@ export abstract class AzExtTreeItem implements types.AzExtTreeItem {
         this._commandId = id;
     }
 
+    public get subscription(): types.ISubscriptionContext {
+        const result = this._subscription || this.parent?.subscription;
+        if (!result) {
+            throw Error(localize('noSubscriptionFound', 'No subscription found for this tree item.'));
```

Comment:
I think it wouldn't hurt to specify Azure subscription for clarity since Static Web Apps also leverage GitHub.